images/build-plugins/on-registry-changed.sh

```diff
@@ -116,7 +116,7 @@ push_index() {
     pushd "$2"
 
     git add index.yaml
-    git commit --message="update(index.yaml): new plugins registry data" --signoff
+    git commit --message="update(index.yaml): new rules registry data" --signoff
```

Comment:
I think this is a small copy/paste error, right? :)

Reply:
:/